### DISCUSSION TITLE
作者简介只有一页

### DISCUSSION THREAD

**Discussion by Nono-404** [note286/xduts#69] [Q&A] [answered]
看word模板这部分应该是单双两页，但latex模板好像只有作者简介页没有空白页，不知道是不是我设置的问题。
![image](https://user-images.githubusercontent.com/56815855/224601586-afebafc7-2db9-404d-98a5-b6cf3a3a72bf.png)

**Comment by note286** [comment, marked as answer]
感谢提醒，已经修复，请从GitHub获取最新版。
